assets/prefabs/Equipment/Steel/SteelSabre.prefab

```diff
@@ -0,0 +1,22 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:SteelSword",
+        "damageType" : "swordDamage",
+        "baseCooldownTime" : 200,
+        "cooldownTime" : 200
+    },
+    "DisplayName" : {
+        "name" : "Steel Sabre",
+        "description" : "A sword with a slightly curved blade without too much weight."
+    },
+    "EquipmentItem" : {
+        "level" : 20,
+        "quality" : 3,
+        "type" : "Melee Weapon",
+        "location" : "Weapon",
+        "attack" : 21,
+        "defense" : 5,
+        "weight" : 10,
+        "speed" : 10
+    }
```

Comment:
Missing } at the very end here (IntelliJ should show a little red squiggly line)